cypress/features/regression/profile.feature

```diff
@@ -1,78 +1,74 @@
 Feature: Profile default component
-  I want to change default Profile component properties
-
-  Background: Open Profile default component page
-    Given I open "Profile" component page
+  I want to test default Profile component properties
 
   @positive
   Scenario Outline: Set email to <email>
-    When I set email to <email> word
+    When I open default "Profile" component in noIFrame with "profile" json from "commonComponents" using "<nameOfObject>" object name
     Then email is set to <email>
     Examples:
-      | email              |
-      | example@email.com  |
-      | johnsmith@sage.com |
+      | email              | nameOfObject |
+      | example@email.com  | email1       |
+      | johnsmith@sage.com | email2       |
 
   @positive
   Scenario Outline: Get avatar via email
-    When I set email to <email> word
+    When I open default "Profile" component in noIFrame with "profile" json from "commonComponents" using "<nameOfObject>" object name
```

Comment:
Can we use `emailGravatar` instead of `<nameOfObject>` ? There is only one example

Reply:
fixed